src/solver/conv_hip_implicit_gemm_bwd_data_v1r1.cpp

```diff
@@ -635,6 +633,14 @@ size_t ConvHipImplicitGemmBwdDataV1R1::GetWorkspaceSize(const ConvolutionContext
 
 bool ConvHipImplicitGemmBwdDataV1R1::IsApplicable(const ConvolutionContext& ctx) const
 {
+/// \todo Fix and remove this workaround.
+/// There are failures with certain configs,
+/// see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/456
+#if 1
```

Comment:
Please use something like ```#if WORKAROUND_GITHUB_456``` so it's easy to grep all workaround in the code

Reply:
Actually, please refer to the JIRA ticket instead of github issue.

Reply:
I believe this is resolved